addons/mail/models/mail_activity.py

```diff
@@ -60,24 +60,26 @@ def default_get(self, fields):
         domain=['&', ('is_mail_thread', '=', True), ('transient', '=', False)],
         help='Specify a model if the activity should be specific to a model'
              ' and not available when managing activities for other models.')
-    default_next_type_id = fields.Many2one('mail.activity.type', 'Default Next Activity',
+    triggered_next_type_id = fields.Many2one('mail.activity.type', string='Trigger',
         domain="['|', ('res_model_id', '=', False), ('res_model_id', '=', res_model_id)]", ondelete='restrict')
-    force_next = fields.Boolean("Trigger Next Activity", default=False)
-    next_type_ids = fields.Many2many(
+    chaining_type = fields.Selection([
+        ('suggest_next_activity', 'Suggest Next Activity'), ('trigger_next_activity', 'Trigger Next Activity')
+    ], string="Chaining Type", required=True, default="suggest_next_activity")
+    suggested_next_type_ids = fields.Many2many(
         'mail.activity.type', 'mail_activity_rel', 'activity_id', 'recommended_id',
         domain="['|', ('res_model_id', '=', False), ('res_model_id', '=', res_model_id)]",
-        string='Recommended Next Activities')
+        string='Suggest')
     previous_type_ids = fields.Many2many(
         'mail.activity.type', 'mail_activity_rel', 'recommended_id', 'activity_id',
         domain="['|', ('res_model_id', '=', False), ('res_model_id', '=', res_model_id)]",
         string='Preceding Activities')
     category = fields.Selection([
         ('default', 'None'), ('upload_file', 'Upload Document')
-    ], default='default', string='Action to Perform',
+    ], default='default', string='Action',
         help='Actions may trigger specific behavior like opening calendar view or automatically mark as done when a document is uploaded')
     mail_template_ids = fields.Many2many('mail.template', string='Email templates')
     default_user_id = fields.Many2one("res.users", string="Default User")
-    default_description = fields.Html(string="Default Description", translate=True)
+    default_description = fields.Html(string="Default Note", translate=True)
```

Comment:
why renaming this ?

Reply:
Because this field references "Note" in mail_activity. I wil rename the field and add this to the migration script ;D